io/disk/lvsetup.py

```diff
@@ -57,6 +57,10 @@ def setUp(self):
         if self.fs_name == 'xfs':
             pkg = 'xfsprogs'
         if self.fs_name == 'btrfs':
+            if distro.detect().name == "rhel" and \
+               int(distro.detect().version) >= 7 and \
+               int(distro.detect().release) >= 4 :
```

Comment:
ah it has the false result for RHEL8 or RHEL8.1 

can below snippt code will work 
 ```
if distro.detect().name == "rhel"  : 
    if int(distro.detect().version) >= 7 :
         self.cancel("btrfs is not supported with RHEL 7.4 onwards")
   if int(distro.detect().version) == 7 and   int(distro.detect().release) >= 4 :
         self.cancel("btrfs is not supported with RHEL 7.4 onwards")
````

Reply:
can be done praveen.   But tricky part is,  few RHEL8 system works (like as in narasimhans system it gave correct values) where as my system and as well yours have weird behavior with RHEl8.0.

Changes can be done thanks!!

@narasimhan-v